### PR TITLE
Tweak spy targets

### PR DESCRIPTION
**Pull request by flappybatpal**
[BALANCE]

## About the PR 
Adds the ocean map head items as some new bounties, port-a-x remotes, the secure briefcase and toxin briefcase, sec helmet and a couple of low tier items for more variety.
Refactors framed head items (HoS medal, RD Diploma etc...) to use a shared parent type that holds a single instance of the item as contents instead of creating/deleting them, this is required for spy bounty logic to find them.

Added item Port-A-Sci remote at the medium reward tier.
Added item HoP's Golden Crayon at the medium reward tier.
Added item HoP's First Bill at the medium reward tier.
Added item MD's Medical License at the medium reward tier.
Added item RD's degree at the medium reward tier.
Added item secure briefcase at the medium reward tier.
Added item toxin briefcase at the medium reward tier.
Added item hypospray at the low reward tier.
Added item judge robe at the low reward tier.
Added item paramedic suit at the low reward tier.
Added item firesuit at the low reward tier.
Added item packet sniffer at the low reward tier.

Added big object morgue at the low reward tier.
Added big object Valuchimp vending machine at the low reward tier.

Increase HoS headset value to 3, on par with Captain and NTSO.

Remove emergency suit, backpack and civilian headset. Half the crew start with these.
Remove Port-A-Sci objective. There is no way to get a new one.

## Why's this needed? 
More items gives more variety, we lost items in the move away from automatically including all subtypes. The new head bounties should result in spies targeting offices more. The briefcases should also be in medium security areas.
Port-A-Sci is a unique item that cannot be replaced and losing it heavily limits A-zone explorers for the round.

## Changelog

```changelog
(u)Flappybat
(+)Added some new spy bounties, most significantly the Head items such as the HoS medal. Removed the Port-A-Sci bounty,
```

### CODE REVIEW

File: code/obj/decal/poster.dm [hunk 1]
```diff
@@ -778,6 +778,12 @@
 
 			// 0 = GLASS, MEDAL 1 = GLASS OFF, MEDAL IN CASE, 2 = GLASS OFF, MEDAL GONE,
 
+			New()
+				..()
+				var/obj/item/clothing/suit/hosmedal/M = new /obj/item/clothing/suit/hosmedal(src.loc)
+				M.desc = src.desc
+				src.contents.Add(M)
```
Comment (pali6) [inline code/obj/decal/poster.dm:785]:
```suggestion
				var/obj/item/clothing/suit/hosmedal/M = new(src)
				M.desc = src.desc
```

File: code/obj/decal/poster.dm [hunk 2]
```diff
@@ -808,13 +814,14 @@
...
 					if (1)
 						playsound(src.loc, "sound/machines/click.ogg", 50, 1)
-						var/obj/item/clothing/suit/hosmedal/M = new /obj/item/clothing/suit/hosmedal()
-						M.desc = src.desc
-						user.put_in_hand_or_drop(M)
-						user.visible_message("[user] takes the medal from the frame.", "You take the medal out of the frame.")
-						src.icon_state = "frame"
-						src.add_fingerprint(user)
-						src.usageState = 2
+						var/obj/item/clothing/suit/hosmedal/M = locate() in src.contents
```
Comment (pali6) [inline code/obj/decal/poster.dm:817]:
```suggestion
						var/obj/item/clothing/suit/hosmedal/M = locate() in src
```

File: code/obj/decal/poster.dm (hunk 1 above)
Comment (pali6) [reply code/obj/decal/poster.dm:785]: Mirror these changes to the other items too (or even better move it to a common parent and reduce duplicated code but only if you feel like doing that).

**Review by pali6 — CHANGES_REQUESTED** [review]

**Comment by Awfullyfishy** [comment]
> Added item handheld vacuum at the low reward tier.

At least add it to janitor cargo crate, janitor vendor or fabs first, as there's rarely more than one on maps and janitors will sob without them.

**Comment by BatElite** [comment]
List of changes seems good to me, the only one I disagree with is the hypospray - they're pretty easy to get from a medical fab, so it's not something you necessarily need to steal.

**Comment by flappybatpal** [comment]
> List of changes seems good to me, the only one I disagree with is the hypospray - they're pretty easy to get from a medical fab, so it's not something you necessarily need to steal.

So, we already have staple gun and circular saw for medical among other somewhat rare stuff. I don't know if it matters to have another?

**Comment by Tarmunora** [comment]
Add portasci remote if you're going to remove portasci from the list

**Comment by flappybatpal** [comment]
Added all the remotes. Took out vacuum as it might be a bit too hard to get a new one.

Remove emergency suit, backpack and civilian headset for being way too common, I'm pretty sure every job starts with an emergency suit in a box. You can get a backpack from the clothing fab. We have a bunch of silly items but you still have to go find them.

**Comment by Tarmunora** [comment]
Mmm, remove sec helmets (far too common, you can just hack a secvend for them iirc), and the non-portasci remotes (portasci remote was specified in particular because it's a "RD" item, as opposed to the other porter remotes which are more of "department" items, if you get what I mean)

**Comment by flappybatpal** [comment]
This is labelled as conflict but VSC found none in the merge and all tests are passing.

**Review by Tarmunora — APPROVED** [review]

**Pull request merged**